orchagent/mirrororch.cpp

```diff
@@ -294,22 +300,55 @@ void MirrorOrch::deleteEntry(const string& name)
     SWSS_LOG_NOTICE("Removed mirror session %s", name.c_str());
 }
 
-bool MirrorOrch::setSessionState(const string& name, MirrorEntry& session)
+void MirrorOrch::setSessionStatus(const string& name, MirrorEntry& session)
```

Comment:
I think this is confusing to have both `setSessionStatus` and `setSessionState` functions. I would keep only the `setSessionStatus` function since status would refer to the 'final state' of the mirror session.

Reply:
'Status' is actually from 'session.status'. State is the all the states including Status.
Let me know if it makes any sense.

---
In reply to: [215803365](https://github.com/Azure/sonic-swss/pull/609#discussion_r215803365) [](ancestors = 215803365)